web/api/formatters/charts2json.c

```diff
@@ -4,6 +4,41 @@
 
 // generate JSON for the /api/v1/charts API call
 
+static inline const char* get_release_channel() {
+    static int use_stable = -1;
+
+    if (use_stable == -1) {
+		char filename[FILENAME_MAX + 1];
+        snprintfz(filename, FILENAME_MAX, "%s/.environment", netdata_configured_user_config_dir);
+        procfile *ff = procfile_open(filename, "=", PROCFILE_FLAG_DEFAULT);
+        if(!ff) {
+            use_stable=1;
+        } else {
+            ff = procfile_readall(ff);
```

Comment:
Add here:

```c
procfile_set_quotes(ff, "'\"");
```

Then then `value` will always be unquoted.

Reply:
Done, thanks.